README.md

```diff
@@ -1,7 +1,12 @@
 # TRENZ.Lib.RazorMail
 ## Templated transactional e-mail using Razor
 
-This is a simple library you can use to write e-mail templates in [Razor syntax](https://learn.microsoft.com/en-us/aspnet/core/mvc/views/razor). That means you write raw HTML, but elevated with C# — you get `@foreach`, `@switch`, and so on, _and_ you get a strongly-typed model for custom data.
+This is a simple library you can use to write e-mail templates in [Razor syntax](https://learn.microsoft.com/en-us/aspnet/core/mvc/views/razor). That means you write raw HTML,
+but elevated with C# — you get `@foreach`, `@switch`, and so on, _and_ you get a strongly-typed model for custom data.
+
+In NuGet, reference either the `TRENZ.Lib.RazorMail.SystemNet` or the `TRENZ.Lib.RazorMail.MailKit` package, depending
+on which `MailSender` backend you prefer. [MailKit](https://github.com/jstedfast/MailKit) is more modern and powerful,
+but `System.Net.Mail` comes built into .NET. There is no need to reference `TRENZ.Lib.RazorMail` directly.
```

Comment:
Should we rename `TRENZ.Lib.RazorMail` to have a suffix like `Abstractions` or `Base`?

Reply:
I was thinking the same. Perhaps `Core`.

Alas, BaGet doesn't support deprecation. So I guess I'll delist it?